RELEASE-NOTES.txt

```diff
@@ -1,3 +1,11 @@
+1.1.0
+=====
+* Introduced a new feature to export and import sites
```

Comment:
There are two things that I am not sure about for this one:

* should we add it to a separate section that says something like "Feature"?
* what PR should I link here? There are many of the PRs that  implemented this feature

@wojtekn what do you think?

Reply:
Sidenote: I looked through other apps that we release but I did not really find any consistency there

Reply:
It's okay to skip PRs in this case. I think it is good as is, or we could simplify it and say 'Added import and export freetaures'.

Reply:
Sounds good, thank you!